docs/2_5_fmu_distribution.adoc

```diff
@@ -84,19 +88,32 @@ The header files `fmi3PlatformTypes.h`, `fmi3FunctionTypes.h` and `fmi3Functions
 
 ===== Directory `binaries` [[binaries-directory]]
 
-A binary FMU must contain the binary files for all supported platforms in this folder.
+====== FMUs with shared libraries [[binaries-directory-shared]]
+
+A shared library binary FMU must contain, for each supported platform, the shared library `<modelIdentifier>.{dll|dylib|so}` and all files which are needed to load this shared library
+in the corresponding platform-specific binary folder.
 To use the binaries of a specific platform, all items in that platform-specific binary folder must be unpacked at the same location as the binary `<modelIdentifier>.{dll|dylib|so}`.
+This is needed since the shared library may search for other files, needed during library load time, relative to its own file location.
+The FMU importer must unpack these files prior to loading the shared library `<modelIdentifier>.{dll|dylib|so}` and these files must stay there until unloading the shared library is finished.
+
+====== FMUs with static libraries [[binaries-directory-static]]
+
+A static library binary FMU must contain, for each supported platform and ABI, the static library `<modelIdentifier>.{a|lib}` and all files which are needed to link with this static library
```

Comment:
Requiring "all files which are needed to link with this static library" to be contained in the FMU would render any FMU illegal that has external dependencies (caused by the static library).